openforcefield/tests/test_molecule.py

```diff
@@ -708,9 +708,9 @@ def test_to_from_iupac(self, molecule):
                               'DrugBank_3726', 'DrugBank_3844', 'DrugBank_3930', 'DrugBank_4161',
                               'DrugBank_4162', 'DrugBank_4778', 'DrugBank_4593', 'DrugBank_4959',
                               'DrugBank_5043', 'DrugBank_5076', 'DrugBank_5176', 'DrugBank_5418',
-                              'DrugBank_5737', 'DrugBank_5902', 'DrugBank_6304', 'DrugBank_6305',
-                              'DrugBank_6329', 'DrugBank_6355', 'DrugBank_6401', 'DrugBank_6509',
-                              'DrugBank_6531', 'DrugBank_6647',
+                              'DrugBank_5737', 'DrugBank_5902', 'DrugBank_6295', 'DrugBank_6304',
```

Comment:
A bigger issue to tackle than in here, but is there documentation about why these lists of molecules are problematic and why?

Reply:
No. This block is a reservoir of technical debt. Dealing with edge cases in OE and RDK is kinda like running a series of seismographs. When they have big releases it's just like "oh, a bunch of stuff behaves differently now", but to actually go and check+fix every road and power line along the fault would be a full-time job in itself. 

Even worse, I'm sure some of the cases that previously violated one of our assumptions are being _fixed_ in some of these updates. But to catch those we'd need a whole matrix of true/fale negatives/positives. So, it's something that a really cheminformatics-inclined person could set up in the future. But we try to avoid the rabbit hole of doing cheminformatics here.

Reply:
I just made a comment in a related issue about aromaticity models, which is probably the reason behind SOME of these.

Reply:
That was over here, FTR: https://github.com/openforcefield/openforcefield/issues/644#issuecomment-666605131